Fix config blocks in classic remoting docs (#26851)
akka/akka: akka-docs/src/main/paradox/remoting.md

}
```

> As you can see in the example above there are five things you need to add to get started:

Reviewer: :)

Reviewer 2: https://youtu.be/1N6OOWtCYQA?t=9

 * Change provider from `local` to `remote`

>  * Disable artery remoting. Artery become the default remoting implementation in `2.6.0`

Reviewer: ```suggestion
 * Disable artery remoting. Artery is the default remoting implementation since `2.6.0`
```

@@@ note

> In order to switch of the logging, set

Reviewer 2: Isn't it to switch the logging `off`?

Author: Must have been tapping x during stand up :)

> In order to switch of the logging, set

Reviewer: ```suggestion
In order to disable the logging, set
```

> `akka.remote.classic.log-remote-lifecycle-events = off` in your

Reviewer 2: None of those debug logging toggles are available for artery?

Author: These ones are only used from Endpoint.scala and Remoting.scala which is classic. I can see we have tests that set this incorrectly so i'll fix those

Reviewer 3: it's only classic

Reviewer 2: Thx for verifying, I didn't know that before.

following setting:

```

> akka.remote.classsic.untrusted-mode = on

Reviewer 2: three s:es

Reviewer 3: average is fine

host name and port pair that is used to connect to the system from the outside. This requires
special configuration that sets both the logical and the bind pairs for remoting.

```

> akka.remote.clasic.netty.tcp {

Reviewer 2: one s

Author: i need to go back to bed
